task_processing/metrics.py

```diff
@@ -0,0 +1,54 @@
+import mock
+
+try:
+    import yelp_meteorite
+    METRICS_ENABLED = True
+except Exception:
+    METRICS_ENABLED = False
+
+TASKPROC_TASK_LAUNCHED_COUNT = 'taskproc_task_launched_count'
+TASKPROC_TASK_FINISHED_COUNT = 'taskproc_task_finished_count'
+TASKPROC_TASK_FAILED_COUNT = 'taskproc_task_failure_count'
+TASKPROC_TASK_KILLED_COUNT = 'taskproc_task_killed_count'
+TASKPROC_TASK_LOST_COUNT = 'taskproc_task_lost_count'
+TASKPROC_TASK_ERROR_COUNT = 'taskproc_task_error_count'
+TASKPROC_TASK_MAX_RETRIES_COUNT = 'taskproc_task_max_retries_count'
```

Comment:
Can we encapsulate all these constants into the mesos executor? I don't feel like these are general metrics.